src/routes/index.tsx

```diff
@@ -22,18 +27,50 @@ export const Routes = () => (
       <AppProvider>
         <Shell>
           <Route path="/admin" nest>
-            <Route path="/products" component={ProductAnalyticsPage} />
+            <Switch>
+              <Route
+                path="/products"
+                component={() => <ProductAnalyticsPage />}
```

Comment:
@npretto that throws a TypeScript error, because it's trying to pass `params` down the component when the component only accepts a `categoryId`

```
Type '(props: Props) => JSX.Element' is not assignable to type 'ComponentType<RouteComponentProps<{}>> | undefined'.
  Type '(props: Props) => JSX.Element' is not assignable to type 'FunctionComponent<RouteComponentProps<{}>>'.
    Types of parameters 'props' and 'props' are incompatible.
      Type 'RouteComponentProps<{}>' has no properties in common with type 'Props'.ts(2322)
index.d.ts(72, 3): The expected type comes from property 'component' which is declared here on type 'IntrinsicAttributes & RouteProps<undefined, "/products">'
(property) RouteProps<undefined, "/products">.component?: React.ComponentType<RouteComponentProps<{}>> | undefined
```